i18n/en.json

```diff
@@ -1652,6 +1656,8 @@
   "admin.team_settings.team_detail.groupsTitle": "Groups",
   "admin.team_settings.team_detail.manageDescription": "Choose between inviting members manually or syncing members automatically from groups.",
   "admin.team_settings.team_detail.manageTitle": "Team Management",
+  "admin.team_settings.team_detail.membersDescription": "A list of users who are currently in the team right now",
```

Comment:
Sorry, @mkraft. I somehow missed this and just saw it while I was doing doc reviews. You are correct and it might read better as: "A list of users who are currently in the team".

I'll attend to it post-release.